Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_7.md

```diff
@@ -0,0 +1,6 @@
+
+#### Mappers
+
+##### CrowdStrike Falcon Mapper
+
+Fixed an issue where some fields did not align with the newest version of CrowdStrike. Some fields have been modified, and others have been deleted as they no longer return from CrowdStrike.
```

Comment:
Shouldn't it be CrowdStrike Falcon.
Fixed an issue where some fields did not align with the newest version of CrowdStrike Falcon. Some fields have been modified, and others have been deleted as they no longer return from CrowdStrike Falcon.